docs/news/upgrades.rst

```diff
@@ -130,6 +130,17 @@ Inventory variable changes
   been redesigned to use YAML lists instead of dictionaries. See
   :ref:`netbase__ref_hosts` for more details.
 
+- The ``resources__group_name`` variable has been removed in favor of using
+  all the groups the current hosts is in. This change has been reflected in the
+  updated variable ``resources__group_templates``.
+  If you need to use a specific group update the ``resources__group_templates``
+  accordingly.   Read the documentation about `resources__templates <https://docs.debops.org/en/master/ansible/roles/debops.resources/defaults-detailed.html#resources-templates>`_
+  for more information on templating with `debops`. Example:
+
+  .. code-block:: none
+
+    resources__group_templates: '{{ (resources__src + "templates/by-group/" + "special_group") | list }}'
```

Comment:
The example is not needed here, since it's a click away through the variable reference.